docs/src/main/paradox/scala/http/routing-dsl/testkit.md

```diff
@@ -5,9 +5,11 @@ For services built with the Routing DSL Akka HTTP provides a dedicated testkit t
 route logic easy and convenient. This "route test DSL" is made available with the *akka-http-testkit* module.
 To use it include the following dependency:
 
+@@@vars
```

Comment:
Yes, however it looks like we have to either patxh Paradox or rewrite the Spray JSON support section to work with this style.